s3parq/fetch_parq.py

```diff
@@ -71,6 +77,35 @@
     Concat dataframes and return
 '''
 
+def fetch_max_partition_value(bucket: str, key: str, partition: str) -> any:
```

Comment:
i might actually *not* use the word fetch here, since fetch has a particular meaning in this library (e.g. grab a dataframe) and really you could just use the word `get` or some other word (lookup?  etc.)

Reply:
Fixed to be get instead of fetch, thats much clearer indeed